canonical-web-and-design/jp.ubuntu.com
# Replace static routes with template finder

## Done

Replace static routes with template finder

## QA

Make sure all these routes work:

https://jp-ubuntu-com-296.demos.haus/openstack
https://jp-ubuntu-com-296.demos.haus/iot
https://jp-ubuntu-com-296.demos.haus/ai-ml
https://jp-ubuntu-com-296.demos.haus/cloudnativedasytokyo2020
https://jp-ubuntu-com-296.demos.haus/kubernetes
https://jp-ubuntu-com-296.demos.haus/enterprise-support
https://jp-ubuntu-com-296.demos.haus/pricing
https://jp-ubuntu-com-296.demos.haus/download
https://jp-ubuntu-com-296.demos.haus/download/thank-you
https://jp-ubuntu-com-296.demos.haus/contact-us
https://jp-ubuntu-com-296.demos.haus/thank-you
https://jp-ubuntu-com-296.demos.haus/engage/openstack-made-easy
https://jp-ubuntu-com-296.demos.haus/engage/yahoo
https://jp-ubuntu-com-296.demos.haus/engage/sbi
https://jp-ubuntu-com-296.demos.haus/engage/robotics_whitepaper
https://jp-ubuntu-com-296.demos.haus/engage/cyberdyne
https://jp-ubuntu-com-296.demos.haus/engage/rigado-casestudy
https://jp-ubuntu-com-296.demos.haus/engage/case-study-gmo-pepabo
https://jp-ubuntu-com-296.demos.haus/engage/redhat-openstack-whitepaper

## Issue / Card

Fixes #295 #209
